packages/edit-site/src/components/layout/index.js

```diff
@@ -195,7 +206,7 @@ export default function Layout() {
 									<Sidebar />
 								</NavigableRegion>
 							</motion.div>
-						) }
+						}
```

Comment:
I think the changes here introduced a regression to the site editor animation. When moving from "edit" to "view" mode, the frame is not animating as expected anymore.

Reply:
Issue here #51903